### PR TITLE
ONE 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/archfiend_of_the_dross.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Archfiend of the Dross
+ManaCost:2 B
+Types:Creature Phyrexian Demon
+PT:6/6
+K:Flying
+K:etbCounter:OIL:4
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigRemoveCtr | TriggerDescription$ At the beginning of your upkeep, remove an oil counter from Archfiend of the Dross. Then if it has no oil counters on it, you lose the game.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/archfiend_of_the_dross.txt:7]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/black_suns_twilight.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Black Sun's Twilight
+ManaCost:X B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature | TargetMin$ 0 | TargetMax$ 1 | NumAtt$ -X | NumDef$ -X | IsCurse$ True | SubAbility$ DBReanimate | SpellDescription$ Up to one target creature gets -X/-X until end of turn. If X is 5 or more, return a creature card with mana value X or less from your graveyard to the battlefield tapped.
+SVar:DBReanimate:DB$ ChangeZone | Origin$ Graveyard | Chooser$ You | ChangeNum$ 1 | ConditionCheckSVar$ X | ConditionSVarCompare$ GE5 | Destination$ Battlefield | Hidden$ True | ChangeType$ Creature.YouOwn+cmcLEX
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/black_suns_twilight.txt:5]: Tapped

File: forge-gui/res/cardsfolder/upcoming/bloated_contaminator.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Bloated Contaminator
+ManaCost:2 G
+Types:Creature Phyrexian Beast
+PT:4/4
+K:Trample
+K:Toxic:1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigProliferate | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or dies, proliferate.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bloated_contaminator.txt:7]: Wrong Description

File: forge-gui/res/cardsfolder/upcoming/encroaching_mycosynth.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Name:Encroaching Mycosynth
+ManaCost:3 U
+Types:Artifact
+S:Mode$ Continuous | Affected$ Permanent.nonLand+YouCtrl | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | AddType$ Artifact | Description$ Nonland permanents you control are artifacts in addition to their other types The same is true for permanent spells you control and nonland permanent cards you own that aren't on the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/encroaching_mycosynth.txt:4]: Missing .

File: forge-gui/res/cardsfolder/upcoming/geth_thane_of_contracts.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Geth, Thane of Contracts
+ManaCost:1 B B
+Types:Legendary Creature Phyrexian Zombie
+PT:3/4
+S:Mode$ Continuous | Affected$ Creature.Other+YouCtrl | AddPower$ -1 | AddToughness$ -1 | Description$ Other creatures you control get -1/-1.
+A:AB$ ChangeZone | Cost$ 1 B B T | Origin$ Graveyard | SorcerySpeed$ True | Destination$ Battlefield | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature in your graveyard | SubAbility$ DBPump | SpellDescription$Return target creature card from your graveyard to the battlefield. It gains "If this creature would leave the battlefield, exile it instead of putting it anywhere else." Activate only as a sorcery. 
+SVar:DBPump:DB$ Pump | LeaveBattlefield$ Exile | Defined$ Targeted | Duration$ Permanent
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/geth_thane_of_contracts.txt:7]: LeaveBattlefield$ doesn't grant the ability, use `Animate...Replacements$`

File: forge-gui/res/cardsfolder/upcoming/archfiend_of_the_dross.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+Types:Creature Phyrexian Demon
+PT:6/6
+K:Flying
+K:etbCounter:OIL:4
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigRemoveCtr | TriggerDescription$ At the beginning of your upkeep, remove an oil counter from CARDNAME. Then if it has no oil counters on it, you lose the game.
+SVar:TrigRemoveCtr:DB$ RemoveCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1 | SubAbility$ LoseGame
+SVar:LoseGame:DB$ LosesGame | Defined$ You | ConditionDefined$ Self | ConditionPresent$ Card.counters_EQ0_OIL
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.OppCtrl | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ Whenever a creature an opponent controls dies, its controller loses 2 life.
+SVar:TrigLoseLife:DB$ LoseLife | LifeAmount$ 2 | Defined$ TriggeredCardController
+DeckHas:Ability$Counters
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/archfiend_of_the_dross.txt:12]: Can the AI play this well? Probably a `AI:RemoveDeck:` of some sort

File: forge-gui/res/cardsfolder/upcoming/black_suns_twilight.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
+Name:Black Sun's Twilight
+ManaCost:X B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select up to one target creature | TargetMin$ 0 | TargetMax$ 1 | NumAtt$ -X | NumDef$ -X | IsCurse$ True | SubAbility$ DBReanimate | SpellDescription$ Up to one target creature gets -X/-X until end of turn. If X is 5 or more, return a creature card with mana value X or less from your graveyard to the battlefield tapped.
+SVar:DBReanimate:DB$ ChangeZone | Origin$ Graveyard | Chooser$ You | ChangeNum$ 1 | ConditionCheckSVar$ X | ConditionSVarCompare$ GE5 | Destination$ Battlefield | Hidden$ True | Tapped$ True | ChangeType$ Creature.YouOwn+cmcLEX
+SVar:X:Count$xPaid
+DeckNeeds:Ability$Graveyard
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/black_suns_twilight.txt:7]: I think this is `DeckHas`

File: forge-gui/res/cardsfolder/upcoming/geth_thane_of_contracts.txt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Name:Geth, Thane of Contracts
+ManaCost:1 B B
+Types:Legendary Creature Phyrexian Zombie
+PT:3/4
+S:Mode$ Continuous | Affected$ Creature.Other+YouCtrl | AddPower$ -1 | AddToughness$ -1 | Description$ Other creatures you control get -1/-1.
+A:AB$ ChangeZone | Cost$ 1 B B T | Origin$ Graveyard | SorcerySpeed$ True | Destination$ Battlefield | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature in your graveyard | SubAbility$ DBAnimate | SpellDescription$ Return target creature card from your graveyard to the battlefield. It gains "If this creature would leave the battlefield, exile it instead of putting it anywhere else." Activate only as a sorcery. 
+SVar:DBAnimate:DB$ Animate | Replacements$ ReplaceLeaves | Defined$ Targeted | Duration$ Permanent
+SVar:ReplaceLeaves:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | ValidCard$ Card.Self | ReplaceWith$ Exile | Description$ If this creature would leave the battlefield, exile it instead.
+SVar:Exile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | Defined$ ReplacedCard
+DeckHints:Ability$Graveyard|Mill|Dredge
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/geth_thane_of_contracts.txt:10]: maybe Sacrifice too?

**Review by Northmoc — APPROVED** [review]
Looks good to me!

**Pull request merged**